docs/components/optimize/userguide/additional-features/variable-labelling.md

```diff
@@ -0,0 +1,20 @@
+---
+id: variable-labeling
+title: Variable Labeling
+description: Add a label for a variable in a process definition.
+---
+
+The variable labeling functionality allows users to add, update and delete batches of variable labels. This allows Optimize to display a variable's label instead of its original name, anywhere the given process definition is being used. Some examples of that would be
```

Comment:
recommend adding the use case
...batches of variable labels **so your data is more understandable by business users**